### ISSUE TITLE
LGPL,beware!

### ISSUE THREAD

**Issue by GoogleCodeExporter** [pyscripter/MustangpeakEasyListView#12]
```
You code is unusable in commercial projects, because LGPL not only requires to 
open library's modifications, but also the possibility for end-user to replace 
the library with a custom build:

effectively this legally means, that the Software Vendor either should link 
dynamically(like runtime packages) OR
 provide object files of HIS OWN (not only the library, but the PROPRIENTARY STUFF) to enable such user to RE-LINK the executable.

---
Bottom line:GPL infected, beware.


```

Original issue reported on code.google.com by `mailtoA...@gmail.com` on 28 Jan 2014 at 4:55

**Comment by pyscripter** [comment]
LICENSE file was added to the front page of the project.   It states the terms specified by the original author, so that potential users can decide whether to use the component or not.
